tox.ini

```diff
@@ -26,6 +27,17 @@ commands =
       --cov-report xml \
       {posargs}
 
+[testenv:cog]
+base_python = python3.13
+skip_install = true
+deps =
+    -r tools_requirements.txt
+    cog
```

Comment:
Maybe it's not `cog` on PyPI? → https://pypi.org/project/cog/

Reply:
Looks like it's cogapp (https://pypi.org/project/cogapp/)

Reply:
Thanks, updated.